app/lib/imports/missing_move_start_events.rb

```diff
@@ -46,6 +46,8 @@ def results
       @current_move = nil
 
       results.save(move, record)
+
+      sleep 3
```

Comment:
:/

Reply:
Haha yes, it's not great... I'm hoping to revert this once the import has been done.